appveyor_mingw_cygwin.yml

```diff
@@ -1,62 +1,36 @@
-init:
-  - echo %PYTHON%
-
-environment:
-  PYTHON: "C:/Python37-x64"
-
-  matrix:
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
-      BUILD: MINGW64
-      INTEGRATION_TESTS: 1
-      ARCHITECTURE: x86_64
-      UNIT_TESTS: 1
-      WEBREADY: False
-      WARNINGS_AS_ERRORS: ON
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
-      BUILD: CYGWIN64
-      INTEGRATION_TESTS: 1
-      ARCHITECTURE: x86_64
-      UNIT_TESTS: 1
-      WEBREADY: False
-      WARNINGS_AS_ERRORS: ON
+image: Visual Studio 2019
```

Comment:
I simplified the AppVeyor file since it now just contains the Cygwin workflow - there are some similarities, but not entirely equivalent platform to MinGW. We plan to move over Cygwin to GHA as well in separate PR after this one.

Reply:
Oh right, thanks for the clarification.